_targets.R

```diff
@@ -22,12 +26,17 @@ list(
 
   # Pull site data
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>% 
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+    names = state_abb,
```

Comment:
⭐ I like that you included `names` here, after `values` is specified, rather than as the final argument, as I think it's nice for the user to see how the targets will be named before they are defined.